TicketSwap/solar
# Combobox V2

# Description

Introducing the new Combobox

## How to test

- Checkout this branch
- `$ yarn && yarn storybook`
- Try it out on mobile and use the arrow keys as well to see if the Combobox is accessible.

## Screenshots

<img width="873" alt="image" src="https://user-images.githubusercontent.com/12168698/203282378-548fcb81-84cc-41c9-b8b3-567c394a9129.png">

## To be notified

@TicketSwap/frontend
